src/components/GalaxyModal.js

```diff
@@ -397,7 +446,7 @@ export const GalaxyModalBase = withDisplayName('GalaxyModal')(
           getEnvMessageBasedOnStatus(app)
         ]),
         div({ style: { paddingBottom: '1.5rem', borderBottom: `1px solid ${colors.dark(0.4)}` } }, [
-          renderGalaxyCostBreakdown(kubernetesRuntimeConfig, dataDiskSize)
+          renderGalaxyCostBreakdown(kubernetesRuntimeConfig, dataDisk.size, dataDisk.diskType)
```

Comment:
We could have a bit more concision by having

```suggestion
          renderGalaxyCostBreakdown(kubernetesRuntimeConfig, dataDisk)
```

if we change the definition of the function as something like

```javascript
const renderGalaxyCostBreakdown = (kubernetesRuntimeConfig, { size, diskType }) => { … }
```